data-serving/scripts/prune-uploads/hooks/country_export.py

```
@@ -92,3 +94,4 @@ def run(sources: list[dict[str, Any]], env: str, dry_run: bool = False):
                 )
         except Exception as e:
             logging.exception(f"Error occurred while trying to submit {jobdef}")
+    return errors
```

I would also `raise` here instead of return the list of errors